packages/core/src/Cardano/util/estimateStakePoolAPY.ts

```diff
@@ -1,26 +0,0 @@
-import { Percent, StakePoolEpochRewards } from '../types';
-
-const MILLISECONDS_PER_DAY = 1000 * 60 * 60 * 24;
-
-/**
- * Estimates annualized percentage yield given past stake pool rewards.
- * Assumes 365 day year.
- *
- * @param rewardsHistory The list of `StakePoolEpochRewards` to estimate the APY
- * @returns `null` if provided an empty `rewardsHistory` or the estimated APY otherwise
- */
-export const estimateStakePoolAPY = (rewardsHistory: StakePoolEpochRewards[]): Percent | null => {
```

Comment:
This util might still be useful for someone, maybe let's keep it?

Reply:
We can keep the `StakePoolEpochRewards` type, just remove `StakePool.epochRewards`. What do you think @rhyslbw ?

Reply:
Yes, just dropping from the provider API seems like the right move here

Reply:
48d72f92f should solve this

Reply:
@iccicci please bring back estimateStakePoolApy with tests too

Reply:
586eb3 should solve this last comment as well